Disable "open this link" confirmation box

This is one of those trends in security that ends up being annoying and nothing
else.

--- tdesktop-5.13.1-full.orig/Telegram/SourceFiles/core/click_handler_types.cpp
+++ tdesktop-5.13.1-full/Telegram/SourceFiles/core/click_handler_types.cpp
@@ -126,7 +126,7 @@
 		const auto parsedUrl = url.startsWith(u"tonsite://"_q)
 			? QUrl(url)
 			: QUrl::fromUserInput(url);
-		if (UrlRequiresConfirmation(parsedUrl) && !base::IsCtrlPressed()) {
+		if (false && UrlRequiresConfirmation(parsedUrl) && !base::IsCtrlPressed()) {
 			const auto my = context.value<ClickHandlerContext>();
 			if (!my.show) {
 				Core::App().hideMediaView();
